crates/ra_lsp_server/src/main_loop/handlers.rs

```diff
@@ -596,6 +598,10 @@ pub fn handle_code_action(
     for source_edit in assists.chain(fixes) {
         let title = source_edit.label.clone();
         let edit = source_edit.try_conv_with(&world)?;
+
+        // We cannot use the 'editor.action.showReferences' command directly
```

Comment:
@kjeremy I think this comment got put in the wrong location, didn't it?

Reply:
Oops. Not sure how that happened